Resources/Prototypes/Kitchen/meal_recipes.yml

```diff
@@ -32,7 +32,7 @@
 
 - type: microwaveMealRecipe
   id: RecipeSuperBiteBurger
-  name: Super Bite Burger Recipe
+  name: super bite burger recipe
```

Comment:
Super Bite Burger sounds like the kind of thing on par with Big Mac, so probably capitals?